backup/Duke.java

```diff
@@ -0,0 +1,116 @@
+import java.io.*;
+import java.util.Scanner;
+
+public class Duke {
+    private static final String FILENAME = "./data/duke.txt";
+    private static final File FILE = new File("./data/duke.txt");
+    private static final String LOGO = " ____        _        \n"
+                                     + "|  _ \\ _   _| | _____ \n"
+                                     + "| | | | | | | |/ / _ \\\n"
+                                     + "| |_| | |_| |   <  __/\n"
+                                     + "|____/ \\__,_|_|\\_\\___|\n";
+
+    // initialise the user and loads data from file to TaskList if file exists.
+    private static void initialise(Scanner scanner) throws FileNotFoundException {
+        String welcomeMsg = "Hello from\n" + LOGO;
+        welcomeMsg += Parser.parse("Hello, I'm Duke! What's your name?");
+        System.out.println(welcomeMsg);
+        String name = scanner.nextLine().strip();
+        if (!FILE.exists() && FILE.length() > 0) { // if file is empty or file does not exist
+            welcomeMsg = Parser.parse("Hello " + name + "! What can I do for you?");
+        } else { // initialise TaskList
+            BufferedReader br = new BufferedReader(new FileReader(FILE));
+            String input;
+            try {
+                while ((input = br.readLine()) != null) {
+//                    System.out.println("hello?");
+                    String[] inputArr = input.split(" | ");
+                    switch (inputArr[0]) {
+                        case "T":
```

Comment:
For the switch case statement, the proper indentation is the word “case” should be the same indent as the word “switch”. Take note.